app/controllers/api_controller.rb

```diff
@@ -120,7 +120,7 @@ def set_access_control_headers
   # not have these. They would instead dealing with the /api/auth
   # mechanism.
   #
-  if Vmdb::Application.config.action_controller.allow_forgery_protection
+  if Vmdb::Application.config.action_controller.allow_forgery_protection && respond_to?(:verify_authenticity_token)
```

Comment:
> @jrafanie the previous / now-LHS conditional was a copy of the one wrapping the protect_from_forgery call... is it still required?

Good question @matthewd. [I guess not](https://github.com/rails/rails/blob/a4ef62279d69d901b99fe0e71338b5aaebab01c0/actionpack/lib/action_controller/metal/request_forgery_protection.rb#L127).  It looks like `protect_from_forgery` enables the `verify_authenticity_token` callback and this was not being called at all with my current change since I removed the `protect_from_forgery`.  So, I guess my solution is not enabling this callback in the normal case.  Hold, while I fix this.